connaisseur/admission_review.py

```diff
@@ -60,6 +62,9 @@ def get_admission_review(
             status["message"] = msg
         review["response"]["status"] = status
 
+    if warnings:
+        review["response"]["warnings"] = warnings
```

Comment:
I think this list would be passed by reference, so whoever receivers the returned `review` can alter the list that is passed into this method!

Solution: make sure to make a copy of the list before you use it, e.g. `warnings = warnings[:]` at the beginning of the method. (This would also favor the default argument to be an empty list, so you don't need to worry about the `None` case.)